evalml/pipelines/components/transformers/preprocessing/datetime_featurizer.py

```diff
@@ -51,8 +53,9 @@ def __init__(self, features_to_extract=None, random_state=0, **kwargs):
                          random_state=random_state)
 
     def fit(self, X, y=None):
-        if not isinstance(X, pd.DataFrame):
-            X = pd.DataFrame(X)
+        X = _convert_to_woodwork_structure(X)
+        X = _convert_woodwork_types_wrapper(X.to_dataframe())
+        # to updateeeeeeee
```

Comment:
Hahaha

Can delete this comment?

Reply:
Yes 😂 

To note though: I meant to update this to use Woodwork but on second thought, might be best to hold off. Woodwork's `select` is not currently organized by the original order of the columns. Maybe doesn't matter in this case but good to keep in mind; will update for #1290